keys.txt

```diff
@@ -20,6 +20,7 @@
 "View player info" 105
 "Toggle fullscreen" 1073741892
 "Toggle fast-forward" 1073741881
+"Fleet: Hyperspace jump" 107
```

Comment:
I'm happy that fleet jump is now a separate command, but I would really like it to be triggered using shift+J instead of a separate key.

Reply:
I see two issues with this approach:
1. It would no longer be independently customizable
2. Shift and J are quite far apart (at least for me, since I use right Shift rarely)

I wouldn't want to do this.

Reply:
For me it feels logical that shift+J is used for fleet-jump if the J-key is used for jumping, similar to shift-R(select nearest ship) relates to the R-key(select nearest hostile).
I would expect the fleet-jump command do be given at rather peaceful moments, so that it is a bit more keypresses than regular J is also fine for me.

Another option is to remove shift completely and do something like this:
- Pressing J starts a normal jump (as is done today).
- Pressing J and waiting for the fleet to jump activates fleet-jump. (keep the effect of waiting not just for the first jump, but propagate it).

Reply:
The game uses quite a number of buttons already, I prefer to keep all jump-related actions under 1 (configurable) key.